src/runtime/runtime.js

```diff
@@ -863,6 +863,12 @@ export class ConfiguredRuntime {
   eventManager() {
     return this.eventManager_;
   }
+
```

Comment:
wasnt this already there

Reply:
Hold off on reviewing this.  It will use logger instead

Reply:
It is ready for review. We decided offline to use EventManager's logEvent method instead of using LoggerApi or Runtime's logEvent function to (1) reduce the number of methods that do logging code and (2) to delineate between external and internal logging by saying "external loggers use LoggerApi, internal loggers use EventManager".